### PR TITLE
[OV][JS] Expose the Tensor.isContinuous to Node.js API

### PR DESCRIPTION
**Pull request by nashez**
### Details:
* Add a TensorWrap::is_continuous function: Calls the underlying Tensor.isContinous function
* Update the addon.ts file with the isContinuous method
* Add unit tests for the isContinuous Api

### Tickets:
Closes: #27701

### CODE REVIEW

**Review by vishniakov-nikolai — APPROVED** [review]
@nashez thank you for contribution!

**Comment by nashez** [comment]
Thanks for the approval @vishniakov-nikolai 

Could you please trigger a CI run for this ?
There might be some uncaught clang format related issues on the PR because my machine doesn't support the clang-format-fixall command.

If there are any issues, I will resolve them along with your existing comment and push later today

**Comment by almilosz** [comment]
build_jenkins

File: src/bindings/js/node/src/tensor.cpp [hunk 1]
```diff
@@ -175,3 +176,12 @@ Napi::Value TensorWrap::get_size(const Napi::CallbackInfo& info) {
     const auto size = static_cast<double>(_tensor.get_size());
     return Napi::Number::New(env, size);
 }
+
+Napi::Value TensorWrap::is_continuous(const Napi::CallbackInfo& info) {
+    Napi::Env env = info.Env();
+    if (info.Length() > 0) {
+        reportError(env, "isContinuous() does not accept any arguments.");
+        return env.Null();
```
Comment (almilosz) [inline src/bindings/js/node/src/tensor.cpp:184]:
```suggestion
        return env.Undefined();
```

Comment (nashez) [reply src/bindings/js/node/src/tensor.cpp]: Done

File: src/bindings/js/node/lib/addon.ts [hunk 2]
```diff
@@ -419,6 +419,10 @@ interface Tensor {
    * It gets the tensor size as a total number of elements.
    */
   getSize(): number;
+  /**
+   * It gets the tensor is continuous or not.
```
[the remark this answers is not in the document]
Comment (nashez) [reply src/bindings/js/node/lib/addon.ts]: Done

**Comment by nashez** [comment]
@almilosz @vishniakov-nikolai Made the requested changes. Please review.

**Comment by nashez** [comment]
@almilosz @vishniakov-nikolai I see the CI failing. 
Is it some issue in the PR or some other issue with the master branch?

**Comment by vishniakov-nikolai** [comment]
> @almilosz @vishniakov-nikolai I see the CI failing. Is it some issue in the PR or some other issue with the master branch?

Hi @nashez! Sometimes it happens, don't worry. I'll take care about it and let you know if some changes in your brunch will be needed.

**Comment by vishniakov-nikolai** [comment]
build_jenkins

**Pull request merged**